en/webserver-setup.md

```diff
@@ -11,82 +14,119 @@
         - [Phalcon configuration](#cherokee-phalcon-configuration)
-    - [Built in Webserver](#php-built-in)
-        - [Phalcon configuration](#php-built-in-phalcon-configuration)
 
 </div>
 
 <a name='setup'></a>
 # Web Server Setup
 In order for the routing of the Phalcon application to work, you will need to set up your web server to process the redirects properly. Setup instructions for popular web servers are:
 
-<a name='nginx'></a>
-## Nginx
-[Nginx](http://wiki.nginx.org/Main) is a free, open-source, high-performance HTTP server and reverse proxy, as well as an IMAP/POP3 proxy server. Unlike traditional servers, Nginx doesn't rely on threads to handle requests. Instead it uses a much more scalable event-driven (asynchronous) architecture. This architecture uses small, but more importantly, predictable amounts of memory under load.
+<a name='php-fpm'></a>
+## PHP-FPM
+The [PHP-FPM](http://php.net/manual/en/install.fpm.php) (FastCGI Process Manager) is usually used to allow the processing of PHP files. Nowadays, PHP-FPM is bundled with all Linux based PHP distributions.
 
-The [PHP-FPM](http://php-fpm.org/) (FastCGI Process Manager) is usually used to allow Nginx to process PHP files. Nowadays, PHP-FPM is bundled with all Linux based PHP distributions. Phalcon with Nginx and PHP-FPM provide a powerful set of tools that offer maximum performance for your PHP applications.
+On **Windows** PHP-FPM is in the PHP distribution archive through the file `php-cgi.exe` and you can start it with this script to help set options. Windows does not support unix sockets so this script will start fast-cgi in TCP mode on port `9000`.
 
-<a name='nginx-phalcon-configuration'></a>
-### Phalcon configuration
-The following are potential configurations you can use to setup Nginx with Phalcon:
+Create the file `php-fcgi.bat` with the following contents:
 
-<a name='nginx-phalcon-configuration-basic'></a>
-#### Basic configuration
-Using `$_GET['_url']` as source of URIs:
+```bat
+@ECHO OFF
+ECHO Starting PHP FastCGI...
+set PATH=C:\PHP;%PATH%
+c:\bin\RunHiddenConsole.exe C:\PHP\php-cgi.exe -b 127.0.0.1:9000
+```
 
-```nginx
-server {
-    listen      80;
-    server_name localhost.dev;
+<a name='php-built-in'></a>
+## PHP Built-In Webserver (For Developers)
+To speed up getting your Phalcon application running in development the easiest way is to use this built-in PHP server. Do not use this server in a production environment. The following configurations for [NGINX](#nginx) and [Apache](#apache) are what you need.
 
-    # This is the folder that index.php is in
-    root /var/www/phalcon/public;
-    index index.php index.html index.htm;
+<a name='php-built-in-phalcon-configuration'></a>
+### Phalcon configuration
+To enable dynamic URI rewrites, without Apache or NGINX, that Phalcon needs, you can use the following router file:
+<a href="https://github.com/phalcon/phalcon-devtools/blob/master/templates/.htrouter.php" target="_blank">.htrouter.php</a>
```

Comment:
I would prefer to allow this link to target a new tab. If you still disagree I will update it.
Thanks